src/components/data-exploration/heatmap/HeatmapPlot.jsx

```diff
@@ -165,7 +165,9 @@ const HeatmapPlot = (props) => {
   }, [louvainClusterCount]);
 
   useEffect(() => {
-    dispatch(updateCellInfo({ cellName: cellHighlight }));
+    if (cellHighlight) {
```

Comment:
What happens when `cellHighlight` turns to `null`? The tooltips for embedding and heatmap should disappear, does that happen if we have this `if (cellHighlight)`?

Reply:
yes, we check if cellHighlight is defined before rendering the heatmap cell info tooltip